chronosphere/registry/registry.go

```diff
@@ -211,10 +211,10 @@ var Resources = mustValidate([]Resource{
 		DryRun: true,
 	},
 	{
-		Name:   "grafana_dashboard",
+		Name:   "classic_dashboard",
 		Entity: "GrafanaDashboard",
```

Comment:
can we create an alias called ClassicDashboard for GrafanaDashboard where it's needed (near the generated file probably). I thought we updated the swagger to always rename, but think we missed a spot, ideally we'd fix that @bg451